Adyen/UI/List/ListItemView.swift

```diff
@@ -67,11 +65,12 @@ public final class ListItemView: UIView, AnyFormItemView {
         }
         
         imageView.imageURL = item?.imageURL
+        imageView.isHidden = item?.iconMode.isHidden == true
```

Comment:
why not just `imageView.isHidden = item?.iconMode.isHidden` ?

Reply:
`item?.iconMode.isHidden` produces an Optional<Bool> (See code above as well)